src/components/organisms/sidebar/sidebar.tsx

```diff
@@ -101,36 +100,35 @@ export const Sidebar = () => {
 					</div>
 
 					<div className="flex flex-col justify-end gap-5">
-						{featureFlags.systemLog ? (
-							<Button
-								className="hover:bg-green-200"
-								onClick={() => toggleLogger(true)}
-								title={t("systemLog")}
+						<Button
+							className="hover:bg-green-200"
+							onClick={() => toggleLogger(true)}
```

Comment:
Maybe we can do here that a click on the button will also close the logger?

Reply:
I was thinking about it, but as for me better stay _only for open_ state

Reply:
Checked with @haimzlato, second click should close the logger.

Reply:
done